front/src/schema/NFTCatalogSchema.ts

```diff
@@ -0,0 +1,20 @@
+export interface NFTCardDto {
```

Comment:
Can we align on some more universal naming convention of interfaces? If we add stuff like `Dto` throughout the interfaces I think that with 30+ of those we won't know what is what.
Maybe we can just have something like `NFTCardData` or `SingleNFTCard`. @nastyakuzusha what do you think?

Reply:
Here we can't because this is exact type of data that comes from backend. We can change prefix of naming, but it should be DTO as ending, because on backend we have a lot of different types of the same data holder, for example NFTCard, NFTCardEntity, NFTCardShortDto, NFTCardExtednedDto etc... Like we can change this namings for now, but in future we will need to rewrite a lot, because we will generate model for front from codegen and namings will changed

Reply:
https://app.clickup.com/t/32h9tyr
But I created a task for this topic
We can discuss it on next meeting